index.d.ts

```diff
@@ -0,0 +1,54 @@
+import { Middleware, Reducer, AnyAction } from "redux";
```

Comment:
Could you elaborate more on this? Redux is currently defined as devDependency of the project. Redux and React can be moved into peerDependencies of project as they're not truly a dependency but more a requirement. There would be no point of using this library without them already installed, right?

Reply:
Yes, that is what I have written, there is a missing peerDependency of redux and consumers need to know which redux version to use so types are working correctly.

Reply:
I think that's a bit tricky, since this library can be consumed with or without redux, for example if you just wanna use the React components utilities.

I don't see currently any option within npm to specify [optional peerDependencies](https://github.com/npm/npm/issues/3066).

Still, if `npm` only warns, we should go for the latest `redux` version, which is `4.x.x`